packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -153,26 +166,22 @@ export function publish(
       autoLoggingID: getOrSetAutoLoggingID(element),
       flowlet,
       isTrusted,
+      /**
+       * eventIndex will be overridden in the subscriber, we are passing in a value here because
+       * Flow requires it but because some al_ui_events will be filtered out before emitting
+       * events to Falco, we can't generate the event_index here
+       */
       eventIndex: 0,
```

Comment:
I know this is already here but in what cases does this happen?  Seems odd to make the subscriber generate it,  technically it happened regardless of if it gets filtered later downstream or not.